actionview/test/template/resolver_shared_tests.rb

```diff
@@ -0,0 +1,85 @@
+module ResolverSharedTests
+  attr_reader :tmpdir
+
+  def run(*args)
```

Comment:
Is this just to do a `setup`?

Reply:
A setup, but we want to run the test within a block. I based this on the same lines in [file_update_checker_shared_tests.rb](https://github.com/rails/rails/blob/master/activesupport/test/file_update_checker_shared_tests.rb#L21-L25)